lib/cli/handle-error.js

```diff
@@ -118,43 +127,68 @@ module.exports = async (exception, options = {}) => {
       ' For debugging logs, run again after setting the',
       ' "SLS_DEBUG=*" environment variable.',
     ].join('');
-    consoleLog(chalk.red(debugInfo));
-    consoleLog(' ');
+    legacy.consoleLog(chalk.red(debugInfo));
+    legacy.consoleLog(' ');
   }
 
   const platform = process.platform;
   const nodeVersion = process.version.replace(/^[v|V]/, '');
-
-  consoleLog(chalk.yellow('  Get Support --------------------------------------------'));
-  consoleLog(`${chalk.yellow('     Docs:          ')}docs.serverless.com`);
-  consoleLog(`${chalk.yellow('     Bugs:          ')}github.com/serverless/serverless/issues`);
-  consoleLog(`${chalk.yellow('     Issues:        ')}forum.serverless.com`);
-
-  consoleLog(' ');
-  consoleLog(chalk.yellow('  Your Environment Information ---------------------------'));
-  consoleLog(chalk.yellow(`     Operating System:          ${platform}`));
-  consoleLog(chalk.yellow(`     Node Version:              ${nodeVersion}`));
-
   const installationModePostfix = await (async () => {
     if (isStandaloneExecutable) return ' (standalone)';
     if (isLocallyInstalled != null) return isLocallyInstalled ? ' (local)' : '';
     return resolveIsLocallyInstalled() ? ' (local)' : '';
   })();
-  consoleLog(
-    chalk.yellow(`     Framework Version:         ${slsVersion}${installationModePostfix}`)
-  );
-  consoleLog(chalk.yellow(`     Plugin Version:            ${sfeVersion}`));
-  consoleLog(chalk.yellow(`     SDK Version:               ${platformClientVersion}`));
-
   const componentsVersion = (() => {
     try {
       return require('@serverless/components/package').version;
     } catch (error) {
       return 'Unavailable';
     }
   })();
-  consoleLog(chalk.yellow(`     Components Version:        ${componentsVersion}`));
-  consoleLog(' ');
+
+  if (command === 'deploy' && serverless) {
+    log.notice(
+      chalk.hex('#fd5750')(
+        `✖ Stack ${serverless.getProvider('aws').naming.getStackName()} failed to deploy\n`
+      )
+    );
+  }
+
+  const chalkSlsGrey = chalk.hex('#8c8d91');
+  writeText(
+    chalkSlsGrey(
+      `Environment: ${platform}, node ${nodeVersion}, framework ${slsVersion}${installationModePostfix}, plugin ${sfeVersion}, SDK ${platformClientVersion}`
+    )
+  );
+  writeText(chalkSlsGrey('Docs:        docs.serverless.com'));
+  writeText(chalkSlsGrey('Support:     forum.serverless.com'));
+  writeText(chalkSlsGrey('Bugs:        github.com/serverless/serverless/issues'));
```

Comment:
It'll be nicer to use one `writeText` (we can input array of messages, and each will be put as new line)

Reply:
good call :+1: